.env.example

```diff
@@ -0,0 +1,2 @@
+REACT_APP_MAKER_FEE=YOUR_MAKER_FEE
+REACT_APP_TAKER_FEE=YOUR_TAKER_FEE
```

Comment:
This is a great idea. Two comments:

- Let's add the other variables: `REACT_APP_RELAYER_URL`, `REACT_APP_RELAYER_NETWORK_ID` and `REACT_APP_FEE_RECIPIENT` (I think I'm not forgetting anything).
- Let's use example values, so that it's easier to know what it's expected.

Reply:
I added them, also I'd added an env variable to set the cache interval